RELEASING.adoc

```diff
@@ -38,15 +38,15 @@ For GA releases this should be configured under the `central` Maven profile.
 </profile>
 ----
 
-. Ensure all Maven commands listed in `.kokoro/stage.sh` script can be executed successfully locally.
-You can add `-Dgpg.skip` in place of the actual GPG key locally.
+. Invoke the Kokoro `prod:cloud-java-frameworks/spring-cloud-gcp/stage` job.
+Note the staging repository ID in the logging output (i.e. `comgooglecloud-1234`).
 
-. Invoke the Kokoro `stage` job.
-Log in to OSS Sonatype and verify the contents of the staged package.
+. Log in to link:https://oss.sonatype.org/[OSS Sonatype] and verify the contents of the staged package. The URL should be similar to https://oss.sonatype.org/content/repositories/{STAGING_ID}/com/google/cloud/
 
-. Invoke to the Kokoko `promote` job.
+. Invoke to the Kokoko `prod:cloud-java-frameworks/spring-cloud-gcp/promote` job.
+Verify the artifacts have been published (https://search.maven.org/search?q=g:com.google.cloud%20AND%20a:spring-cloud-gcp, may take a few hours).
 
-. Make sure that the link:https://github.com/spring-io/start.spring.io/blob/master/start-site/src/main/resources/application.yml[Spring Boot Initializr] is updated to support the new version of Spring Cloud GCP.
+. Update start.spring.io by ensuring link:https://github.com/spring-io/start.spring.io/blob/master/start-site/src/main/resources/application.yml[Spring Boot Initializr] is updated to include the new version of Spring Cloud GCP.
 
 == Uploading Javadocs
```

Comment:
Does this step really require a branch to have been created, if you're publishing from a release tag?
Speaking of the release tag, we need a step somewhere in the instructions for creating a release tag and updating the changelog.
Also, we need a step for incrementing the project version post-release.

Reply:
Branch? This is for any new version AFAICT. As in 2.0.0 --> 2.0.1 is a release that requires this step, but assuming the branch is `2.0.x` no new branch is needed.

As for branches - We briefly discussed following Spring's approach of using the minor version as the primary increment, but I don't think we committed to it. 

Your comments all point to the desire for a big checklist of steps to go through for each release. While I'm not opposed to that, I haven't ran through a whole release yet, so a fully fleshed out list would be mostly conjecture at this point.

Reply:
We will need a branch for each of the Spring Cloud release trains.
I assume that we will be working on the Ilford release train for a while before jumping into the J____ release train sometime next year. So, we won't really need a `2.x` or `2.0.x` while `master` is aligned with Ilford.

Regarding release tags, we are currently missing them for RC1 and RC2. See: https://github.com/GoogleCloudPlatform/spring-cloud-gcp/releases. Also notice, Step 5 in [R2DBC repo](https://github.com/GoogleCloudPlatform/cloud-spanner-r2dbc/blob/master/RELEASING.adoc).

Reply:
> we won't really need a 2.x or 2.0.x while master is aligned with Ilford.

So if I'm following your train of thought here, once Ilford is GA, the Spring branch will move on to `2021.0.0-SNAPSHOT` or whatever the J release train is. We will continue to use `2020.0.0` on our `master` until we upgrade to `2021.0.0-M1`, at which point we'll make a `2.0.x` branch and start emitting `2.1.0-SNAPSHOT` on `master`. Does that sound about right?

Reply:
> we are currently missing them for RC1 and RC2

RC1 now has a tag, and once RC2 is out I'll tag it too.

Reply:
> > we won't really need a 2.x or 2.0.x while master is aligned with Ilford.
> 
> So if I'm following your train of thought here, once Ilford is GA, the Spring branch will move on to `2021.0.0-SNAPSHOT` or whatever the J release train is. We will continue to use `2020.0.0` on our `master` until we upgrade to `2021.0.0-M1`, at which point we'll make a `2.0.x` branch and start emitting `2.1.0-SNAPSHOT` on `master`. Does that sound about right?

Exactly!